test_jsonschema.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2022 Axis Communications AB.
+# For a full list of individual contributors, please see the commit history.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import subprocess
+
+import jsonschema
+import pytest
+
+
+@pytest.mark.parametrize(
+    "filename",
+    subprocess.check_output(["git", "ls-files", "--exclude-standard", "schemas/*.json"])
+    .decode("utf-8")
+    .splitlines(),
+)
+def test_json_schema(filename):
+    with open(filename) as input_file:
+        data = input_file.read()
+
+    event_schema = json.loads(data)
```

Comment:
Nit: The file that inspired you to write this one needs to retain `data`, but we're only interested in the deserialized JSON so this is shorter and might have a measurable performance impact given the number of files that needs to be processed.

```suggestion
    with open(filename) as input_file:
        event_schema = json.load(input_file)
```